app/models/cda/base_model.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'active_resource'
+module Cda
+  class BaseModel < ActiveResource::Base
+    self.site = CourtDataAdaptor::Resource::V2.api_url
+    self.include_format_in_path = false
+
+    def self.headers
+      base_headers = { 'Content-Type' => 'application/json' }
```

Comment:
This method is a slightly complicated way of creating a hash. Why not just do:

```ruby
{ 
  'Content-Type' => 'application/json',
  'Authorization' => "Bearer #{CourtDataAdaptor::Client.new.bearer_token}"
}
```

That way you don't need to make and modify a variable.

Reply:
I agree, the reason is there was a conditional before, which I removed:

```
if test
   base_headers['Authorization'] = "Bearer #{CourtDataAdaptor::Client.new.bearer_token}" 
end
```